### PR TITLE
PXP-663: [CLI] Depreciate old error handling

### PR DESCRIPTION
**Pull request by mfauzaan**
## Summary
This pull request addresses an issue where the v1 error handling format was accidently replaced with a new one. To ensure consistency across different API environments, a quick switch has been implemented in the CLI. The primary objective is to align the error code format uniformly, irrespective of the API environment.

Additionally, please note that a subsequent pull request will be submitted to clean up the deprecated or old error handling code, maintaining codebase hygiene.

Ticket: https://mindvalley.atlassian.net/browse/PXP-663

### Changed

- [x] Replaced the error match function to enforce the usage of the new error code format only.

### Next Steps 
A follow-up pull request is planned to conduct a thorough cleanup of the outdated error handling code

### CODE REVIEW

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/185#" title="Author self-approved">mfauzaan</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by onimsha — APPROVED** [review]

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/185#" title="Author self-approved">mfauzaan</a>*, *<a href="https://github.com/mindvalley/wukong-cli/pull/185#pullrequestreview-1830901217" title="Approved">onimsha</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
